client/me/help/help-contact/index.jsx

```diff
@@ -177,6 +183,16 @@ const HelpContact = React.createClass( {
 		this.clearSavedContactForm();
 	},
 
+	prepareDirectlyWidget: function() {
+		if (
+			this.hasDataToDetermineVariation() &&
+			this.getSupportVariation() === SUPPORT_DIRECTLY &&
```

Comment:
Random thought for the future:

I wonder if it would be useful to add a `SUPPORT_UNKNOWN` or `SUPPORT_LOADING` variation which is used while we don't yet have all the data needed to determine the variation. That could eliminate the `shouldShowPreloadForm` function.